.github/workflows/release-tools.yml

```
@@ -39,33 +27,43 @@ env:
     runs-on: ubuntu-latest
     steps:
       - name: 📥 Checkout Code
         uses: actions/checkout@34e114876b0b11c390a56381ad16ebd13914f8d5 # v4
         with:
           token: ${{ secrets.THUNDER_AUTOMATION_BOT }}
 
+      - name: 🔀 Resolve Release Channel
+        id: channel
+        env:
+          BRANCH: ${{ github.ref_name }}
+        run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2209:warning:6:3: Use var=$(command) to assign output (or quote to assign string)

(shellcheck)